packages/api-explorer/src/components/DocSDKs/DocSDKs.tsx

```diff
@@ -77,7 +77,7 @@ export const DocSDKs: FC<LanguageSDKProps> = ({ api, method, type }) => {
                 : gen.declareType('', item as IType)
               return (
                 <TabPanel key={language}>
-                  <DocCode language={language} code={code} />
+                  <DocCode language={language} code={code} editor="prism" />
```

Comment:
Perhaps the prop should say `editor="ace"` for places where we are still using ace so that it's easier to find and replace later on. This is completely fine as is though, just throwing a suggestion out there.